src/cpp/types/topic/DatabrokerTopic.cpp

```diff
@@ -65,9 +65,24 @@ bool DatabrokerTopic::operator <(
     }
     else
     {
-        return topic_type_.compare(other.topic_type_);
+        // To equal name, compare type
+        // WARNING: do not return value from compare, as -1 != false
+        if (topic_type_.compare(other.topic_type_) < 0)
+        {
+            return true;
+        }
+        else
+        {
+            return false;
+        }
     }
 }
 
+std::ostream &operator<<(std::ostream &os, const DatabrokerTopic& a)
+{
+    os << "{<" << a.topic_name() << ">:<" << a.topic_type() << ">}";
```

Comment:
I'd remove the `<>` as it indicates that there should be something else there.
```suggestion
    os << "{" << a.topic_name() << ":" << a.topic_type() << "}";
```

Reply:
I though about it, but there are topics and especially types that use `:` i.e. `std::msg::string`

Reply:
Okay so maybe lets use
```cpp
os << "{" << a.topic_name() << ", " << a.topic_type() << "}";
```

Reply:
:+1: